bindings_node/src/permissions.rs

```diff
@@ -161,12 +161,13 @@ pub struct PermissionPolicySet {
   pub update_group_description_policy: PermissionPolicy,
   pub update_group_image_url_square_policy: PermissionPolicy,
   pub update_group_pinned_frame_url_policy: PermissionPolicy,
+  pub update_message_expiration_ms_policy: PermissionPolicy,
```

Comment:
I Added only one policy for both `message_expiration_ms` and `message_expiration_from_ms`. wdyt?

Reply:
yea I think that makes sense, doesnt make sense for someone to be able to change one without the other 👍